cmadison/cmadison.py

```diff
@@ -135,11 +135,14 @@ def download(self):
 
         try:
             resp = requests.get(url)
-            resp.raise_for_status()
-            with open(self.fname, 'wb+') as fd:
-                for chunk in resp.iter_content(chunk_size=128):
-                    fd.write(chunk)
-            return True
+            if resp.status_code == 404:
```

Comment:
The resp.raise_for_status() will raise an HTTPError if the status_code is in the 4xx or 5xx range. Its supposed to be caught on line 146, but the problem is that the wrong HTTPError is being used here.

The code is importing the HTTPError from urllib but the error raised is requests.exceptions.HTTPError. I think the right fix is to change the import.